templates/common/on-prem/units/nodeip-configuration.service.yaml

```diff
@@ -35,6 +35,7 @@ contents: |
     --prefer-ipv6 \
     {{end -}}
     --retry-on-failure \
+    ${NODEIP_HINT:-} \
```

Comment:
Grepping the repo I see we do `${NODEIP_HINT:-${KUBELET_NODEIP_HINT:-}}; \`. Which one do you want here, yours or the one that repo already has?

Reply:
I noted that in the PR description: "This deliberately does not support the old KUBELET_NODEIP_HINT name that the UPI service does because it was never a thing in IPI and we don't need the backward compatibility with it."